Sources/StreamChat/Query/ChannelListQuery.swift

```diff
@@ -46,12 +46,6 @@ extension Filter where Scope: AnyChannelListFilterScope {
     }
 }
 
-// We don't want to expose `members` publicly because it can't be used with any other operator
-// than `$in`. We expose it publicly via the `containMembers` filter helper.
```

Comment:
Shouldn't we expose something like `equalMembers(userIds:)` since only $eq and $in operator are supported?

Reply:
It feels more closer to public docs if we allow using it directly. But I am lacking some historic knowledge on channel list filters so not sure what is best here.

Reply:
With other keys we can create incompatible combinations.

Reply:
Yeah, my guess is that for some properties, we started limiting the operators, especially for performance reasons. But for other properties we did not follow the same approach.

IMO, it would be better that programmatically you could only use the operators that you are allowed to, and that it would not require you to look into docs to see which operators are supported. It would be better to just limit every field but now it is kinda too late, since we already expose most of them :/ 

So yeah, maybe it is better we open up this one for consistency

Reply:
Since we have that containMembers, maybe it is better then to have equalMembers. Hmm